text/0002-keyboard-accessibility.md

```diff
@@ -0,0 +1,50 @@
+- Start Date: March 16, 2023
+- RFC PR: (leave this empty)
+- Issue: (leave this empty)
+
+# Summary
+
+We need a keyboard accessibility standard for Webstudio, since global standards only provide recommendations in many cases, but we need to be consistent.
```

Comment:
> since global standards only provide recommendations in many cases

I think this is a bit unfair. ARIA Authoring Practices is more than a recommendation, and is in fact required as part of [Accessible Rich Internet Applications (WAI-ARIA)](https://www.w3.org/TR/wai-aria/#managingfocus). The problem is that folks think accessibility stops at WCAG and don't know that other standards exist.

Does ARIA Authoring Practices cover every possible interaction? No, it lacks guidelines for canvases. But it covers a good 80% IMO. In the past their standard widgets were quite hard to find and the website was ugly. However the new version ARIA Authoring Practices 1.2 is much easier to read, use, and understand IMO.

`[For example here is an ARIA Authoring Practices 1.1 example](https://www.w3.org/TR/2018/NOTE-wai-aria-practices-1.1-20180726/examples/checkbox/checkbox-1/checkbox-1.html)`, [compared to the same one in 1.2](https://www.w3.org/WAI/ARIA/apg/patterns/checkbox/examples/checkbox/).

[I highly recommend giving it another read through and staying within these patterns when using the appropriate widget.](https://www.w3.org/WAI/ARIA/apg/patterns/)